src/Behat/Gherkin/Node/FeatureNode.php

```diff
@@ -72,6 +77,7 @@ public function __construct(
         $description,
         array $tags,
         BackgroundNode $background = null,
+        array $rules,
```

Comment:
I have a slight concern that we lose the ordering of scenarios+rules here by having them as separate arguments. (The main concern is the impact on Behat's formatters).

Perhaps we should be passing an array of Scenarios+Rules? Needs some thought